lib/web/join_tokens.go

```diff
@@ -52,6 +52,8 @@ import (
 const (
 	teleportOSSPackageName = "teleport"
 	teleportEntPackageName = "teleport-ent"
+
+	cloudStableChannelRepo = "cloud/stable"
```

Comment:
The channel name is `stable/cloud`, not `cloud/stable`.
```suggestion
	cloudStableChannelRepo = "stable/cloud"
```